OrbitLinuxTracing/PerfEventOpen.cpp

```diff
@@ -77,6 +77,20 @@ int sample_event_open(uint64_t period_ns, pid_t pid, int32_t cpu) {
   return generic_event_open(&pe, pid, cpu);
 }
 
+int sample_event_fp_open(uint64_t period_ns, pid_t pid, int32_t cpu) {
```

Comment:
Rename to sample_event_callchain_open (similar reasons as above).

Reply:
Thanks for the suggestion. I renamed it accordingly.

Reply:
Don't split "event" and "open", no other method here does it. Put "callchain" in front.